modules/account-lib/src/coin/sol/iface.ts

```diff
@@ -0,0 +1,40 @@
+import { Blockhash, SystemInstructionType, TransactionSignature } from '@solana/web3.js';
+import { InstructionBuilderTypes } from './constants';
+
+// TODO(STLX-9890): Add the interfaces for validityWindow, feeOptions and SequenceId
+export interface SolanaKeys {
```

Comment:
should be singular: `SolanaKey` since this represents a single key.

Reply:
Is similar to DefaultKeys, it contains a key pair, priv and pub